packages/next/src/build/babel/loader/transform.ts

```diff
@@ -86,6 +86,9 @@ export default function transform(
     target,
     filename,
   })
+  if (!babelConfig) {
```

Comment:
In what case is this hit? When there is no babel config it shouldn't add this loader 🤔

Reply:
`experimental.reactCompiler: true` && `filename.includes('/node_modules/')` for now

I found this edge case after adding the check for node modules.
In the future, files filtered by https://github.com/vercel/next.js/pull/75605 will also hit this line.